kafka-rest/pom.xml

```diff
@@ -141,17 +141,17 @@
         </dependency>
         <dependency>
             <groupId>org.bouncycastle</groupId>
-            <artifactId>bcprov-jdk15on</artifactId>
```

Comment:
jdk 1.5 is probably older than us, so we should use jdk 1.8 at least :)

Reply:
Do we state anywhere our minimum JRE version?  If we do and it's 8, then all good.

If we do and it's 5 (which would be a surprise) then we have a problem.

If we don't, then it's probably worth a quick readme update to add a line to say java 8 and later.

Reply:
https://docs.confluent.io/platform/current/installation/versions-interoperability.html#java

Java 8 is min, and its going to be deprecated soon.